UIKitTemplate/UIKitTemplate/Controllers/NotificationViewController.swift

```diff
@@ -0,0 +1,140 @@
+// NotificationViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Контроллер с уведомлениями
+final class NotificationViewController: UIViewController {
+    // MARK: - RowsType
+
+    enum RowsType {
+        case notificationPicture
+        case notificationButton
+    }
+
+    // MARK: - Constants
+
+    private enum Constants {
+        static let notificationPictureCellIdentefire = "NotificationPicture"
+        static let notificationButtonCellIdentefire = "NotificationButton"
+        static let headeSectionOne = "Сегодня"
+        static let headerSectionTwo = "На этой неделе"
+    }
+
+    // MARK: - Visual Components
+
+    private lazy var notificationTableView: UITableView = {
+        let tableView = UITableView()
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.register(
+            NotificationPictureCell.self,
+            forCellReuseIdentifier: Constants.notificationPictureCellIdentefire
+        )
+        tableView.register(
+            NotificationButtonCell.self,
+            forCellReuseIdentifier: Constants.notificationButtonCellIdentefire
+        )
+        tableView.separatorStyle = .none
+        tableView.allowsSelection = false
+        tableView.rowHeight = UITableView.automaticDimension
+        tableView.showsVerticalScrollIndicator = false
+        return tableView
+    }()
+
+    // MARK: - Private Properties
+
+    private let typeRows: [RowsType] = [.notificationPicture, .notificationButton]
+
+    private var rMLinkStorage = RMLinkStorage()
+
+    // MARK: - Life Cycle
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureUI()
+    }
+
+    // MARK: - Private Methods
+
+    private func configNavigation() {
```

Comment:
```suggestion
    private func configurateNavigation() {
```
 лучше выбери себе конкретные наименования для методов , лучше полные а не сокращенные и их используй везде одинаковые. а то у тебя где то confi, где то config